recipes/mesas/meta.yaml

```diff
@@ -3,19 +3,20 @@
 # Jinja variables help maintain the recipe as you'll update the version only here.
```

Comment:
Please copy the example, rather than removing it

Reply:
Thankyou! Fixed